dissect/cstruct/tools/stubify.py

```diff
@@ -0,0 +1,105 @@
+# Searches and creates a stub of a cstruct definitions
```

Comment:
I'd put this as the description (maybe with a few more words) of the argument parser rather than a comment here.